spec/aptible/cli/subcommands/db_spec.rb

```diff
@@ -93,17 +98,37 @@ class Account < OpenStruct
   end
 
   def setup_prod_and_staging_accounts
-    staging_redis = Database.new(handle: 'staging-redis-db')
-    staging_postgres = Database.new(handle: 'staging-postgres-db')
-    prod_elsearch = Database.new(handle: 'prod-elsearch-db')
-    prod_postgres = Database.new(handle: 'prod-postgres-db')
+    staging_dbs = []
+    prod_dbs = []
+    staging_dbs << Database.new(handle: 'staging-redis-db')
+    staging_dbs << Database.new(handle: 'staging-postgres-db')
+    prod_dbs << Database.new(handle: 'prod-elsearch-db')
+    prod_dbs << Database.new(handle: 'prod-postgres-db')
+
+    stub_local_token_with('the-token')
+    setup_new_accounts_with_dbs(
+      token: 'the-token',
+      account_db_mapping: {
+        'staging' => staging_dbs,
+        'production' => prod_dbs
+      }
+    )
+  end
+
+  def setup_many_prod_and_staging_accounts(count)
```

Comment:
This isn't really necessary... ignore this. I need a spec that stubs `Aptible::API::Database.all`.